tensorflow/core/kernels/batching_util/adaptive_shared_batch_scheduler.h

```diff
@@ -425,7 +425,7 @@ void AdaptiveSharedBatchScheduler<TaskType>::MaybeScheduleNextBatch() {
     return;
   }
   auto best_it = batches_.end();
-  double best_score;
+  double best_score = (std::numeric_limits<double>::max)();
```

Comment:
Why are the parenthesis around `max` needed? I thought the problem with the old PR was simply the missing parenthesis at the end.

Reply:
> Why are the parenthesis around `max` needed? I thought the problem with the old PR was simply the missing parenthesis at the end.

Ref the discussion on [MSDN](https://social.msdn.microsoft.com/Forums/vstudio/en-US/e94865f7-84cd-4912-8339-6ee95fb58825/numericlimitsltdoublegtmax-not-recognized?forum=vcgeneral) and [stackoverflow](https://stackoverflow.com/questions/27442885/syntax-error-with-stdnumeric-limitsmax),

"The <Windows.h> header file that includes macro definitions named max and min:
`#define max(a,b) (((a) > (b)) ? (a) : (b))`
The preprocessor replaces the max identifier in the expression:
`std::numeric_limits<size_t>::max()` to `std::numeric_limits<size_t>::(((a) > (b)) ? (a) : (b))`

Wrap the call to max with parenthesis, which prevents the macro expansion.
" by Piotr Skotnicki